examples/decoding/plot_decoding_spatio_temporal_source.py

```diff
@@ -44,7 +49,7 @@
 
 # Setup for reading the raw data
 raw = io.read_raw_fif(raw_fname, preload=True)
-raw.filter(2, None, fir_design='firwin')  # replace baselining with high-pass
+raw.filter(0.1, None, fir_design='firwin')
```

Comment:
In my experience it's not necessary to high-pass if we baseline, but I just kept the example here;

The main issue is that high passing can spread artifacts; the main advantage is that it doesn't hide design problem (e.g. this dataset is not randomized IIRC, and you can get higher than chance level before t0)